Stripe/STPPaymentContext.m

```diff
@@ -21,6 +21,19 @@
 #import "STPPaymentConfiguration+Private.h"
 
 #define FAUXPAS_IGNORED_IN_METHOD(...)
+#define FAUXPAS_IGNORED_IN_CLASS(...)
```

Comment:
Yeah, it seems like people have just been pasting them around, but I might make a new PR to just add all the fauxpas headers in a single file.